package.json

```diff
@@ -72,6 +72,8 @@
   },
   "dependencies": {
     "form-urlencoded": "1.2.0",
-    "querystring": "0.2.0"
+    "lodash": "4.17.4",
```

Comment:
I meant that the dependency here should be [`lodash.get`](https://www.npmjs.com/package/lodash.get). Since we're not using webpack here, there's no tree-shaking so we're still getting all of `lodash`.